backend/src/main/java/me/performancereservation/domain/performance/entities/PerformanceSchedule.java

```diff
@@ -24,15 +24,22 @@ public class PerformanceSchedule extends BaseEntity {
 
     private int remainingSeats; // 남은 좌석
 
-    private boolean is_canceled; // 회차 취소 여부
+    @Column(name = "is_canceled")
+    private boolean canceled; // 회차 취소 여부
 
     @Builder
-    public PerformanceSchedule(Long id, Long performanceId, LocalDateTime startTime, LocalDateTime endTime, int remainingSeats, boolean is_canceled) {
+    public PerformanceSchedule(Long id, Long performanceId, LocalDateTime startTime, LocalDateTime endTime, int remainingSeats, boolean canceled) {
         this.id = id;
         this.performanceId = performanceId;
         this.startTime = startTime;
         this.endTime = endTime;
         this.remainingSeats = remainingSeats;
-        this.is_canceled = is_canceled;
+        this.canceled = canceled;
+    }
+
+    public void cancel() {
+        if (!this.canceled) {
+            this.canceled = true;
+        }
     }
```

Comment:
```suggestion
    public void cancel() {
       if (this.canceled) {
            throw ErrorCode.ALREADY_CANCLED_SCHEDULE.domainException("이미 취소된 공연입니다.");
        }
        this.canceled = true;
    }
```

이런식으로 이미 취소된 공연은 중복으로 취소못하도록 예외처리를 추가하는 것도 좋아보여요

Reply:
저희가 DDD를 따르기 보단 서비스 중심으로 개발을 하자라고 생각했을 때 도메인에는 간단한 필드의 상태를 바꾸는 정도만 메서드로 두고 예외나 추가적인 로직은 서비스에서 다 수행하는 게 좋을 것 같아서 도메인에는 간단한 상태 변경만 뒀었습니다.

Reply:
팀 컨벤션과 좋아하는 취향 차이기도 한 것 같아요 ㅋㅋㅋ 저도 1차 프로젝트 하기전까진 광현님 스타일이었습니다 ㅋㅋㅋ